esp-box/esp-box.c

```diff
@@ -368,5 +367,14 @@ esp_err_t bsp_button_init(const bsp_button_t btn)
 
 bool bsp_button_get(const bsp_button_t btn)
 {
-    return !(bool)gpio_get_level(btn);
+    if (btn == BSP_BUTTON_MAIN) {
+        uint8_t home_btn_val = 0x00;
+        assert(tp);
+
+        esp_lcd_touch_get_button_state(tp, 0, &home_btn_val);
```

Comment:
Do we need to wrap this part with `#ifdef CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0`?

Alternatively, `_Static_assert(CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0, "Touch buttons must be supported for this BSP");`

Reply:
I agree, there should be `#ifdef` or `#warning`.